### PR TITLE
Quartermaster Drip

### PR DESCRIPTION
**Pull request by Christasmurf**
## What Does This PR Do
Resprites the quartermaster uniform and skirt to match what I ported for the HoP, as I plan to do with the other heads too.
Adds a formal, dress and whimsy uniform for the QM
Adds coat and mantle suit slot items for the QM
Adds cap and beret head items for the QM

## Why It's Good For The Game
Better sprites, more variety, etc

## Images of changes
Before:
image

After:
image

image
image

## Testing
Above images + fucked around with a magic mirror making sure to check EVERY item individually on EVERY species

## Changelog
:cl:
add: Adds formal, dress and whimsy uniforms; coat and mantle suit slot items; cap and beret hats for the quartermaster.
imageadd: Resprites the base quartermaster jumpsuit and jumpskirt
/:cl:

### CODE REVIEW

**Comment by meow20** [comment]
These are so amazing, god damn one of best sprite changes i've seen in ages, old ones really sucked.

File: code/modules/clothing/suits/job_suits.dm [hunk 1]
```diff
@@ -280,6 +280,29 @@
 		"Kidan" = 'icons/mob/clothing/species/kidan/suit.dmi'
 	)
 
+//Quartermaster
+/obj/item/clothing/suit/mantle/qm
+	name = "quartermaster's mantle"
+	desc = "An armoured shawl for the quartermaster. Keeps the breeze from the vents away from your neck."
```
Comment (Henri215) [inline code/modules/clothing/suits/job_suits.dm:286]:
```suggestion
	desc = "An armored shawl for the quartermaster. Keeps the breeze from the vents away from your neck."
```

File: code/modules/clothing/suits/job_suits.dm [hunk 2]
```diff
@@ -280,6 +280,29 @@
...
+	icon_state = "qmmantle"
+	item_state = "qmmantle"
+	allowed = list(/obj/item/paper, /obj/item/clipboard, /obj/item/gun/energy/kinetic_accelerator, /obj/item/melee/baton, /obj/item/flashlight/seclite, /obj/item/melee/classic_baton/telescopic, /obj/item/melee/knuckleduster, /obj/item/rcs)
+	armor = list(MELEE = 15, BULLET = 10, LASER = 15, ENERGY = 5, BOMB = 15, RAD = 0, FIRE = 50, ACID = 50)
+
+/obj/item/clothing/suit/qmcoat
+	name = "quartermaster's coat"
+	desc = "A brown trenchcoat to show the station you mean business. Has armour woven within the fabric."
```
Comment (Henri215) [inline code/modules/clothing/suits/job_suits.dm:294]:
```suggestion
	desc = "A brown trenchcoat to show the station you mean business. Has armor woven within the fabric."
```

**Review by Henri215 — COMMENTED** [review]
image

You moved these or something? They are kidan clothing?

File: code/modules/clothing/head/job_hats.dm [hunk 3]
```diff
@@ -72,6 +72,28 @@
...
+	)
+
+/obj/item/clothing/head/qm/beret
+	name = "quartermaster's beret"
+	desc = "A brown beret with a golden cargo insignia emblazoned on it. Rule over Cargonia in style."
+	icon_state = "beret_qm"
+	dog_fashion = /datum/dog_fashion/head/qm
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/head.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/head.dmi',
```
Comment (Henri215) [inline code/modules/clothing/head/job_hats.dm:94]:
```suggestion
		"Drask" = 'icons/mob/clothing/species/drask/head.dmi'
```

**Comment by SynthTwo** [comment]
I LOVE YOU CHRISTA

**Comment by Christasmurf** [comment]
> image
> 
> You moved these or something? They are kidan clothing?

Brand new file, kidan didn't previously have cargo clothing. Those are human clothes so that kidan doesn't get invisible outfits when wearing those items.

**Review by Henri215 — APPROVED** [review]
Looking honkable to me

**Comment by Danchi299** [comment]
Old suit made them look like any cargo technician
New suit makes them look like they are Head of Personnel
I suppose paperwork pals should look alike

**Comment by Christasmurf** [comment]
> Old suit made them look like any cargo technician New suit makes them look like they are Head of Personnel I suppose paperwork pals should look alike

As I get further through the outfits it will become clearer, but the Command staff are going to have the same base uniform while the rest of the station with have a different style (command with shirt/trousers and the rest of crew having a full jumpsuit)

The secondary outfits will vary depending on role, but the main outfits will all be either the lower-crew style or command style.

**Review by S34NW — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**